doc/classes/EditorProperty.xml

```diff
@@ -134,6 +134,14 @@
 				Emit it if you want to key a property with a single value.
 			</description>
 		</signal>
+		<signal name="property_pinned">
+			<argument index="0" name="property" type="StringName" />
+			<argument index="1" name="pinned" type="bool" />
+			<description>
+				Emit it if you want to mark (or unmark) a the value of a property for being saved regardless being equal to the default.
```

Comment:
```suggestion
				Emit it if you want to mark (or unmark) the value of a property for being saved regardless of being equal to the default.
```

Reply:
```suggestion
				Emit if you want to mark (or unmark) a property to be saved regardless if it's equal to the default value.
```

Reply:
Coming in next push (a mix of both; the same for 3.x).